provider/config.rb

```diff
@@ -29,6 +29,7 @@ class Config < Api::Object
     attr_reader :tests
     attr_reader :files
     attr_reader :changelog
+    # TODO(rileykarson): update this
```

Comment:
I think this is probably worth updating in this PR.  I see that you said this isn't done, but I think the documentation change belongs in the first PR.  If you aren't sure about that, I'd prefer you delete the entire comment and leave "TODO(rileykarson): document this when #1055 is done" or something like that.

Reply:
It doesn't look like it's used at all anymore! Removing altogether- I didn't think this would be the case yet.

If this errors out, I'll revert that & update this to match reality.